assets/js/blocks/reviews-by-product/style.scss

```diff
@@ -34,121 +35,139 @@
 	}
 
 	.wc-block-reviews-by-product__orderby {
-		margin-bottom: $gap-larger;
+		margin-bottom: $gap-small;
 		text-align: right;
 	}
 
 	.wc-block-reviews-by-product__orderby__label {
 		margin-right: $gap-small;
+		display: inline-block;
+		font-weight: normal;
 	}
 
 	.wc-block-reviews-by-product__list {
-		list-style: none;
+		list-style: none outside;
 		margin: 0;
 	}
 
 	.wc-block-reviews-by-product__item {
-		margin-bottom: $gap-larger;
+		margin: 0 0 $gap-large * 2;
+		list-style: none outside;
```

Comment:
That declaration is inherited from the `.wc-block-reviews-by-product__list`, so it shouldn't be necessary to declare it here too.

Reply:
Shouldn't but is. Some themes declare bullet styles at li level.

Reply:
In that case, could we remove it from `.wc-block-reviews-by-product__list`?

`list-style` is only applied to `<li>` elements, so if we are not inheriting it, there is no reason to set it for the `<ul>` tag.